cmake/Modules/FindRTIConnextDDS.cmake

```diff
@@ -350,13 +350,13 @@
 #   ``CONNEXTDDS_IMPORTED_TARGETS_DEBUG`` will not have any effect.
 #
 # Note
-# ^^^^
+# ----
 # Some flags related to the compiler, like (-std=c++11 needed when linking
```

Comment:
I think you mean this:

Some flags related to the compiler, (such as -std=c++11, needed when linking